### DISCUSSION TITLE
A1601 VAPIX commands not found issue

### DISCUSSION THREAD

**Discussion by KenPixelMii** [AxisCommunications/discussions#132] [VAPIX]
Hello, 

I'm new to Axis and VAPIX so please excuse my noob-ness. I am part of a team building a mobile security application where we need to interact with an Axis A1601. We have a test controller all set up and we can reach the web gui from both the LAN and the WAN. 

Initially the hardware was running a firmware version of 1.84.4 and we had **successfully** used the API call to "axtdc:GetDoorConfigurationList" to return the device configuration response.  

During a call with Axis tech support for a controller config issue they required us to upgrade firmware to 11.1.10.1 and now all of our attempts to call the API are returning a "404 Not Found". 

These are example requests that worked for us previously which now returns a 404: 

`curl "https://<ip>/vapix/doorcontrol/axtdc:GetDoorConfigurationList" --digest -u <username>:<pass> -s`

`curl "https://<ip>/vapix/doorcontrol/tdc:UnlockDoor" --digest -u <username>:<pass> -s -d’{"Token": "Axis-b8a44f7c482f:1669821849.533926000"}’`

It seems that /vapix is not even in the path now, hence the 404. Does the 11.1.10.1 firmware not include support for VAPIX? If so, what are our options? 

Thank you in advance for any assistance.

**Comment by onie5** [comment]
Hello Ken,

VAPIX is only available for firmware version 1.84. When upgrading to version 10.x and above, VAPIX is disabled as there are other functions made for use with Secure Entry in Axis Camera Station. So if you need VAPIX, keep at 1.84

**Reply by KenPixelMii** [reply]
Thank you for the quick response! I'm getting so much conflicting information. I attempted to roll back the firmware to 1.84 and essentially bricked the controller. Called into Axis Tech Support and they told me 1) you can't downgrade from any firmware passed 10.x to 1.84 and 2) that VAPIX works on 11.1.10.4 and that the Camera Station even uses it. This seems hard to believe as I don't see any trace of VAPIX in this version. I can't seem to understand why everything is so siloed - as soon as I mention VAPIX the Tech Support team gets immediately into a rant about how they don't support it and to ask the Developer Community or consult the VAPIX Library documentation which is out of date. I really want to work with this controller but these things are making it very difficult. 

Any advice would be appreciated. 
Ken

**Reply by onie5** [reply]
Hello Ken,

Have you tried defaulting the controller after downgrading it? If it still does not work you may need to send it for RMA.

I cannot comment on the support engineer’s comment on VAPIX on fw 11 but your best bet would be the release notes. It is correct that the controller will work with Axis Camera Station and that is the reason why VAPIX is not working. You can read release notes for 10.1 which mentions no 3rd party interfacing will work and no changes from there yet.
